pages/overview.md

```diff
@@ -1,44 +1,35 @@
 
+The [National Ecological Observatory Network (NEON)](https://www.neonscience.org/) provides unprecedented, standardized ecological data across the United States, including systematic collections of carabid beetles ([DP1.10022.001](https://data.neonscience.org/data-products/DP1.10022.001)) from diverse terrestrial ecosystems ([Thorpe et al., 2016]( https://doi.org/10.1002/ecs2.1627)). These collections, housed in the [NEON Biorepository](https://biorepo.neonscience.org), include high-resolution images of individual specimens. This challenge leverages this unique resource to explore a novel approach to environmental monitoring.
 
 ## Setup Overview
-This challenge is designed to simulate a real-world biological scenario. Suppose a biologist studies a particular butterfly Species A with many subspecies. One day, the biologist finds that a subset of the images collected looks slightly abnormal in their visual appearance. The biologist does not recognize the pattern as belonging to any of the subspecies on which their research is focused. After investigation, the biologist finds that these unusual samples are hybrids produced by different subspecies of Species A. Realizing that they may encounter other hybrids in future collections of images, the biologist seeks an anomaly detection algorithm to automatically identify (unseen) hybrid cases.
+The goal of this challenge is to predict the drought status at a location on a specific date based on images of carabid beetle specimens collected at the location on that date. This will be an out-of-sample challenge because training data will be assembled from one site of locations, and test data will assembled from a different set of locations. All locations will be NEON sites.
```

Comment:
```suggestion
The goal of this challenge is to predict the drought status at a location on a specific date based on images of carabid beetle specimens collected at the location on that date. This will be an out-of-sample challenge because training data will be assembled from one site of locations, and test data will be assembled from a different set of locations. All locations will be NEON sites.
```